### ISSUE TITLE
Implement UX/UI Changes for SimExchange deposit/withdraw collateral

### ISSUE THREAD

**Issue by travisdmathis** [MARKETProtocol/dApp#251]
### Description
Implement new designs and functionality for `exchange/wallet`.

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Feature

### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)
- [ ] New UI/UX has been implemented as per designs and instructions

### Additional Information
[comment]: # (Any other information that would be useful, content, screenshots, etc.)
Blocked By: #250

**Comment by travisdmathis** [comment]
Closing in favor of main issue
